Explorer/Assets/DCL/Navmap/NavmapController.cs

```diff
@@ -86,6 +87,7 @@ public NavmapController(
 
             zoomController = new NavmapZoomController(navmapView.zoomView, dclInput);
             filterController = new NavmapFilterController(this.navmapView.filterView, mapRenderer, webBrowser);
+            new CategoryFilterController(this.navmapView.categoryToggles, mapRenderer);

```

Comment:
It doesnt seem to be necessary to couple it with this class, since it doesnt have any usage. Also, in case of need it, please inject it instead of creating it implicitly here. (I already made a refactor of this class injecting all required controllers)

Reply:
Done!